<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAudioInput</class>
 <widget class="QWidget" name="MyAudioInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyAudioInput</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>160</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTime">
      <property name="text">
       <string>Idle : 0/S</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pButtonRecord">
        <property name="text">
         <string>录音</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pButtonPlay">
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pButtonStopRecord">
        <property name="text">
         <string>停止录音</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pButtonStopPlay">
        <property name="text">
         <string>停止播放</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
